internal/mapping.go

```diff
@@ -128,6 +128,27 @@ func GetSpannerCols(conv *Conv, srcTable string, srcCols []string) ([]string, er
 	return spCols, nil
 }
 
+func getUniqueKeyName(srcKeyName string, schemaKeys map[string]bool) string {
```

Comment:
Consider renaming this as following:

func getUniqueId(srcId string, spannerIds map[string]bool) string

This will allow you to re-use this function further for tableName, columnName in addition to indexKeyName and foreignKeyName.

Also, it's a good idea to move this utility function to the end of the file, such that public functions are listed first in the file.

Reply:
updated name of getUniqueKeyName